socorro/analysis/correlations/correlations_app.py

```diff
@@ -42,20 +29,27 @@
         "socorrolib.lib.transform_rules.TransformRuleSystem",
         "apply_all_rules",
         "socorro.analysis.correlations.core_count_rule"
-            ".CorrelationCoreCountRule, "
+        ".CorrelationCoreCountRule, "
         "socorro.analysis.correlations.interesting_rule"
-            ".CorrelationInterestingModulesRule,"
+        ".CorrelationInterestingModulesRule,"
         "socorro.analysis.correlations.interesting_rule"
-            ".CorrelationInterestingModulesVersionsRule,"
+        ".CorrelationInterestingModulesVersionsRule,"
         "socorro.analysis.correlations.interesting_rule"
-            ".CorrelationInterestingAddonsRule,"
+        ".CorrelationInterestingAddonsRule,"
         "socorro.analysis.correlations.interesting_rule"
-            ".CorrelationInterestingAddonsVersionsRule,"
+        ".CorrelationInterestingAddonsVersionsRule,"
     ],
 ]
 correlation_rule_sets_as_string = json.dumps(correlation_rule_sets)
 
 
+def date_with_default_yesterday(value):
+    if not value:
+        return datetime.datetime.utcnow().date() - datetime.timedelta(days=1)
+    y, m, d = [int(x) for x in value.split('-')]
+    return datetime.date(y, m, d)
```

Comment:
Looks pretty solid to me. It's never supposed to be any other format. Isodate is for when you have unpredictable formats coming in.